tests/phpthemis/composer-setup.sh

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash -e
 
+EXPECTED_SIGNATURE="$(wget -q -O - https://composer.github.io/installer.sig)"
 php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
-php -r "echo 'Installed hash = '; echo hash_file('SHA384', 'composer-setup.php');"
+ACTUAL_SIGNATURE="$(php -r "echo hash_file('sha384', 'composer-setup.php');")"
```

Comment:
is there any difference to use low case? _(just curious)_

Reply:
> is there any difference to use low case?

Not really. [PHP docs](https://www.php.net/manual/en/function.hash-file.php) use lowercase. [The implementation](https://github.com/php/php-src/blob/master/ext/hash/hash.c#L88) converts input to lowercase which is used in the lookup table (_yes, I was curious too_). The case does not matter here.

Reply:
heheh, I like your curiosity level :)